commit/merkleroot/rmn/types/config.go

```diff
@@ -0,0 +1,78 @@
+package types
+
+import (
+	"crypto/ed25519"
+
+	mapset "github.com/deckarep/golang-set/v2"
+
+	cciptypes "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
+)
+
+type NodeID uint32
+
+// RMNHomeConfig contains the configuration fetched from the RMNHome contract.
+type RMNHomeConfig struct {
+	Nodes          []RMNHomeNodeInfo
+	MinObservers   map[cciptypes.ChainSelector]uint64
+	ConfigDigest   cciptypes.Bytes32
+	OffchainConfig cciptypes.Bytes // Raw offchain configuration bytes
+}
+
+// RMNHomeNodeInfo contains information about a node from the RMNHome contract.
+type RMNHomeNodeInfo struct {
+	ID                        NodeID // ID is the index of this node in the RMN config
+	PeerID                    cciptypes.Bytes32
+	SupportedSourceChains     mapset.Set[cciptypes.ChainSelector]
+	SignObservationsPublicKey *ed25519.PublicKey // offchainPublicKey
```

Comment:
Let's rename this to maybe just `OffchainPublicKey` and add a comment e.g `The private key of this pub key.. was used to signed observations...`